variables.tf

```diff
@@ -515,3 +515,8 @@ variable "deck_version" {
   default = "1.0.0"
 }
 
+variable "db_final_snapshot_identifier" {
+  description = "identifier"
```

Comment:
Can you provide a more complete description here? Thanks for the contribution!

Reply:
@DennoVonDiesel done.